src/JewelryDetail.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import request from 'superagent';
+import Spinner from './Spinner.js';
+
+export default class Detail extends Component {
+    state = {
+        jewelry: {},
+    }
+    componentDidMount = async () => {
+
+        this.setState({ loading: true });
+
+        const data = await request.get(`https://katilyn-fullstack-project.herokuapp.com/jewelry=${this.props.match.params.id}`);
```

Comment:
This will need to match your API endpoint. It should a / between the resource and the id. Ex. https://katilyn-fullstack-project.herokuapp.com/jewelry/1 :)